docs/installation/linux/ubuntulinux.md

```diff
@@ -119,25 +119,23 @@ packages from the new repository:
 - Ubuntu Trusty 14.04 (LTS)
 
 For Ubuntu Trusty, Wily, and Xenial, it's recommended to install the
-`linux-image-extra` kernel package. The `linux-image-extra` package
+`linux-image-extra-*` kernel packages. The `linux-image-extra-*` packages
 allows you use the `aufs` storage driver.
 
-To install the `linux-image-extra` package for your kernel version:
+To install the `linux-image-extra-*` packages:
 
 1. Open a terminal on your Ubuntu host.
 
 2. Update your package manager.
 
         $ sudo apt-get update
 
-3. Install the recommended package.
+3. Install the recommended packages.
 
-        $ sudo apt-get install linux-image-extra-$(uname -r)
+        $ sudo apt-get install linux-image-extra-$(uname -r) linux-image-extra-virtual
 
 4. Go ahead and install Docker.
 
-If you are installing on Ubuntu 14.04 or 12.04, `apparmor` is required.  You can install it using: `apt-get install apparmor`
```

Comment:
It's not that it is no longer required, it's automatically installed because of the package dependencies.  Doesn't make sense to list apparmor if we aren't going to list any others, especially since they're automatically installed:
```
# apt-get install -s docker-engine
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following extra packages will be installed:
  apparmor aufs-tools cgroup-lite git git-man iptables libapparmor-perl
  libapparmor1 libedit2 liberror-perl libltdl7 libnfnetlink0
  libsystemd-journal0 libx11-6 libx11-data libxau6 libxcb1 libxdmcp6 libxext6
  libxmuu1 libxtables10 openssh-client patch rsync xauth xz-utils
Suggested packages:
  apparmor-profiles apparmor-docs apparmor-utils gettext-base git-daemon-run
  git-daemon-sysvinit git-doc git-el git-email git-gui gitk gitweb git-arch
  git-bzr git-cvs git-mediawiki git-svn ssh-askpass libpam-ssh keychain
  monkeysphere ed diffutils-doc openssh-server
Recommended packages:
  ssh-client
The following NEW packages will be installed:
  apparmor aufs-tools cgroup-lite docker-engine git git-man iptables
  libapparmor-perl libapparmor1 libedit2 liberror-perl libltdl7 libnfnetlink0
  libsystemd-journal0 libx11-6 libx11-data libxau6 libxcb1 libxdmcp6 libxext6
  libxmuu1 libxtables10 openssh-client patch rsync xauth xz-utils
0 upgraded, 27 newly installed, 0 to remove and 7 not upgraded.
```